launch/robots/robot_panda.launch.py

```diff
@@ -43,8 +47,11 @@ def generate_declared_arguments() -> List[DeclareLaunchArgument]:
         # Model for Ignition Gazebo
         DeclareLaunchArgument(
             "model",
-            default_value="panda",
-            description="Name or filepath of model to load.",
+            default_value=path.join(
+                get_package_share_directory("panda_description"),
+                "panda",
+                "model.sdf",
+            ),
```

Comment:
Is this necessary? The path was intended to be updated to include this model via this CMake hook: https://github.com/AndrejOrsula/panda_ign_moveit2/blob/be5704c5b8ede1ae3998db8285473e6da300cfc0/panda_description/cmake/environment_hooks/sdf_path.bash

Reply:
I was a bit suprised cause everything looked like it should have been working at some time, though i could not ifgure why :)
Had not seen the script or environment hooks prior, will have to check it out. Maybe it's because I'm using zsh and I'd have to have a corresponding zsh hook there. Will try to check